### PR TITLE
Staging - Updated repository with all methods created

### PR DESCRIPTION
**Pull request by mrtoluadesina**

### CODE REVIEW

File: alternatingCase/alternatingCase.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+var toUpper = require('../toUpper/toUpper');
+var toLower = require('../toLower/toLower');
+
+String.prototype.alternatingCase = function() {
+  var newString = '';
+  for (var i = 0; i < this.length; i++) {
+    (i % 2 === 0) ? newString += this[i].toLower() : newString += this[i].toUpper();
```
Comment (mogbeyi-david) [inline alternatingCase/alternatingCase.js]: I suggest you add a comment on this line to explain what the line of code is achieving

File: inverseCase/inverseCase.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+var toUpper = require('../toUpper/toUpper');
+var toLower = require('../toLower/toLower');
+
+String.prototype.inverseCase = function() {
+  var newString = '', regex = /[a-z]/;
```
Comment (mogbeyi-david) [inline inverseCase/inverseCase.js]: Please carry out one declaration per line

File: inverseCase/inverseCase.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  for (var i = 0; i < this.length; i++) {
+    regex.test(this[i]) ? newString += this[i].toUpper() : newString += this[i].toLower();
```
Comment (mogbeyi-david) [inline inverseCase/inverseCase.js]: Add a comment here to explain what the code base is doing, or maybe actually use an if statement to carry out this operation

File: isDigit/isDigit.test.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+var isDigit = require('./isDigit');
+
+test('Checks that a number is a digit - only one number - and returns true', function() {
+  var trueDigit = '3', falseDigit = '34';
```
Comment (mogbeyi-david) [inline isDigit/isDigit.test.js]: Carry out one declaration per line

File: numberWords/numberWords.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  3: 'three',
+  4: 'four',
+  5: 'five',
+  6: 'six',
+  7: 'seven',
+  8: 'eight',
+  9: 'nine'
+}
+String.prototype.numberWords = function() {
+  var regex = /\d/g, number = this.match(regex), strip = /\s$/, result = '';
```
Comment (mogbeyi-david) [inline numberWords/numberWords.js]: Carry out one declaration per line

File: numberWords/numberWords.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+  if (regex.test(this)) {
```
Comment (mogbeyi-david) [inline numberWords/numberWords.js]: Try to refactor the codebase so that you don't need the else statement

File: numberWords/numberWords.js [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+    for (var i = 0; i < number.length; i++) {
+      if (numberInWords.hasOwnProperty(number[i])) {
```
Comment (mogbeyi-david) [inline numberWords/numberWords.js]: Add descriptive comments at this section of the codebase to explain what the codebase is doing

File: toLower/toLower.js [hunk 8]
```diff
@@ -0,0 +1,11 @@
+String.prototype.toLower = function() {
+  var newString = '';
+  for (var i = 0; i < this.length; i++) {
+    code = this.charCodeAt(i);
+    if (code >= 65 && code <= 90) { code += 32; }
```
Comment (mogbeyi-david) [inline toLower/toLower.js]: In trying to implement one-liners, remove the braces from the if-statement

File: toLower/toLower.test.js [hunk 9]
```diff
@@ -0,0 +1,6 @@
+var toLower = require('./toLower');
+
+test('Checks that a string is converted to the lower case value', function() {
+  var aString = 'I OWN DECAGON';
```
Comment (mogbeyi-david) [inline toLower/toLower.test.js]: Oh really! I didn't know that :)

File: toUpper/toUpper.js [hunk 10]
```diff
@@ -0,0 +1,11 @@
+String.prototype.toUpper = function() {
+  var newString = '';
+  for (var i = 0; i < this.length; i++) {
+    code = this.charCodeAt(i);
+    if (code >= 97 && code <= 122) { code -= 32; }
```
Comment (mogbeyi-david) [inline toUpper/toUpper.js]: Remove the braces from the one-liner

File: ucFirst/ucFirst.js [hunk 11]
```diff
@@ -0,0 +1,5 @@
+String.prototype.ucFirst = function() {
+  return String.fromCharCode(this.charCodeAt(0) & 223) + this.substr(1);
```
Comment (mogbeyi-david) [inline ucFirst/ucFirst.js]: This line of code is not really readable as the reader would have to evaluate it in his brain before arriving at what the code base does. Split it to several lines and add descriptive comments in the function.

File: words/words.test.js [hunk 12]
```diff
@@ -0,0 +1,6 @@
+var words = require('./words');
+
+test('Returns individual words in a string', function() {
+  var thisWord = 'I want to make sure this comes back as an array';
+  expect(thisWord.words()).toEqual(expect.arrayContaining(["I", "want", "to", "make", "sure", "this", "comes", "back", "as", "an", "array"]));
```
Comment (mogbeyi-david) [inline words/words.test.js]: Please use single quotes for strings

File: alternatingCase/alternatingCase.test.js [hunk 13]
```diff
@@ -0,0 +1,6 @@
+var alternatingCase = require('./alternatingCase');
```
Comment (mogbeyi-david) [inline alternatingCase/alternatingCase.test.js]: Create a separate folder for test files and put all your test files inside that folder, not in the same folders as your code implementations.

File: fromCurrency/fromCurrency.js [hunk 14]
```diff
@@ -0,0 +1,5 @@
+String.prototype.fromCurrency = function() {
+  return Number((this).replace(/\,/g, '')).toFixed(2);
```
Comment (mogbeyi-david) [inline fromCurrency/fromCurrency.js]: Please add comments on this line

File: isQuestion/isQuestion.js [hunk 15]
```diff
@@ -0,0 +1,6 @@
+String.prototype.isQuestion = function() {
+  var regex = /.+\?$/
```
[the remark this answers is not in the document]
Comment (mrtoluadesina) [reply isQuestion/isQuestion.js]: Done.

File: toLower/toLower.test.js (hunk 9 above)
Comment (mrtoluadesina) [reply toLower/toLower.test.js]: Now you know... :)

File: ucFirst/ucFirst.js (hunk 11 above)
Comment (mrtoluadesina) [reply ucFirst/ucFirst.js]: But you said to achieve it with a one-liner

File: alternatingCase/alternatingCase.test.js (hunk 13 above)
Comment (mrtoluadesina) [reply alternatingCase/alternatingCase.test.js]: What if I just create on universal test file and then import all my methods there and then create all the tests in that file

**Pull request closed without merging**